CoffeeMachine.java

```diff
@@ -0,0 +1,90 @@
+package Lesson_12;
+
+import java.util.Scanner;
+
+public class CoffeeMachine {
+    private String esspresso = "Esspresso";
+    private String macchiato = "Macchiato";
+    private String cappuccino = "Cappuccino";
+    private String milk = "Milk";
+    private String hotChocolate = "Hot Chocolate";
+    private String drinkChoice;
+    private int answer;
+    private char choice;
+    private String[] drinkList = {"Espresso", "Macchiato", "Cappuccino"};
+
+    public CoffeeMachine(){
```

Comment:
This constructor does not serve much of a purpose since outside caller of this class (creator of the class object, cannot pass any parameters while calling this constructor.

When you say: 
```
this.esspresso = esspresso;
```
it is the same as you said:
```
espresso = espresso
```

Since you already have default value for this variable, it is setting the same value again.

If you want to have constructor which will initialize your attributes, you should use:
```
public CoffeeMachine(String esspresso, String macchiato ... etc.) {
 ...
}
```